packages/ember-metal/lib/computed.js

```diff
@@ -564,14 +558,42 @@ ComputedPropertyPrototype.teardown = function(obj, keyName) {
   client.get('fullName'); // 'Betty Fuller'
   ```
 
+  You can pass a hash with two functions, `get` and `set`, as an
+  argument to provide both a getter and setter:
+
+  ```js
+  let Person = Ember.Object.extend({
+    firstName: 'Betty',
```

Comment:
lets put these in `init`.

init() {
  this._super(...arguments)
  this.firstName = 'Betty';
  this.lastName =  'Jones';
}
``